dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -309,28 +433,68 @@ def test_options_words_update(self):
         self.assertDictEqual(expected_word_count, text_profile.word_count)
         self.assertDictEqual(expected_vocab, text_profile.vocab_count)
 
-    def test_options_vocab_update(self):
-        # change vocab.is_enabled, other options remain the same as default values
+    def test_options_most_common_chars_count(self):
+        options = TextProfilerOptions()
+        options.top_k_chars = 3
+
+        text_profile = TextProfiler("Name", options=options)
+        sample = pd.Series(["this is test,", " this is a test sentence",
+                            "this is", "this"])
+        text_profile.update(sample)
+        profile = text_profile.profile
+
+        expected_vocab_count = {'s': 10, 't': 9, ' ': 8}
+        self.assertDictEqual(expected_vocab_count, profile["vocab_count"])
+
+        # change number of common chars
+        options.top_k_chars = 2
+        text_profile = TextProfiler("Name", options=options)
+        text_profile.update(sample)
+        profile = text_profile.profile
+
+        expected_vocab_count = {'s': 10, 't': 9}
+        self.assertDictEqual(expected_vocab_count, profile["vocab_count"])
 
+        # change number of common chars greater than length of vocab_counts list
+        options.top_k_chars = 300
+        text_profile = TextProfiler("Name", options=options)
+        text_profile.update(sample)
+        profile = text_profile.profile
+
+        expected_vocab_count = {'s': 10, 't': 9, ' ': 8, 'i': 7, 'e': 5,
+                                'h': 4, 'n': 2, ',': 1, 'a': 1, 'c': 1}
+        self.assertDictEqual(expected_vocab_count, profile["vocab_count"])
+
+    def test_options_most_common_words_count(self):
         options = TextProfilerOptions()
-        options.vocab.is_enabled = False
+        options.top_k_words = 3
```

Comment:
I think for this and char need to also check the `None` value as that is an option.

Reply:
That can be after the update.

Reply:
added the test for None